oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/measure/BanyanDBMetadataQueryDAO.java

```diff
@@ -162,7 +162,9 @@ protected void apply(MeasureQuery query) {
                         List<AbstractCriteria> instanceRelationsQueryConditions = new ArrayList<>(instanceIds.size());
                         for (final String instanceId : instanceIds) {
                             final IDManager.ServiceInstanceID.InstanceIDDefinition def = IDManager.ServiceInstanceID.analysisId(instanceId);
-                            and(Lists.newArrayList(eq(InstanceTraffic.SERVICE_ID, def.getServiceId()), eq(InstanceTraffic.NAME, def.getName())));
+                            instanceRelationsQueryConditions.add(
+                                and(Lists.newArrayList(eq(InstanceTraffic.SERVICE_ID, def.getServiceId()), eq(InstanceTraffic.NAME, def.getName())))
+                            );
```

Comment:
What are these about?

Reply:
From code, instanceRelationsQueryConditions just init by `new ArrayList<>(instanceIds.size())`, it does not add any criteria later.

Reply:
@lujiajing1126 Could you check this part?

Reply:
@weixiang1862 Nice catch! Thx